docker.go

```diff
@@ -32,6 +34,7 @@ import (
 	"github.com/magiconair/properties"
 	"github.com/moby/term"
 	specs "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/shirou/gopsutil/v3/mem"
```

Comment:
I couldn't find a more reliable way of getting the memory. However, I'm unsure if we really need to log the memory. Also, the package brings with it a few more dependencies which I would prefer to avoid.

Reply:
See my inline comment. We need to know the memory of the Docker host, not of the test process host. A good idea would be to check, how we do it in `testcontainers-java`.